pkg/snmp/traps/listener.go

```diff
@@ -21,7 +21,6 @@ type TrapListener struct {
 	packets       PacketsChannel
 	listener      *gosnmp.TrapListener
 	errorsChannel chan error
-	errorLogger   *log.ThrottledLogger
```

Comment:
Using `util/log` instad of `ThrottledLogger` might lead to too verbose logging.

Maybe similar to tracelog.SetLogger(corelogger{}), we can call `tracelog.SetLogger(corelogger{})`. Since `corelogger` is private, we might need to redeclare something like: https://github.com/DataDog/datadog-agent/blob/c51ff676ad8937ae5e1dd5fc63e4b3c9660fdbf0/cmd/trace-agent/run.go#L236-L281

Also, I think we should ideally not depend on `/pkg/trace`. I'm not sure, if APM team expect other teams to use `ThrottledLogger`.

Reply:
How about switching log level WARN to DEBUG, not using `ThrottledLogger`?
I am wondering if Line 83 need not be WARN because it has not been output since it was introduced.

Reply:
I think this should work, switching to DEBUG log level. Could you make this change?

Reply:
https://github.com/DataDog/datadog-agent/pull/13975/commits/ac47133da8cdf3dedf24299c816482b3b231e87a